dev/bots/run_fuchsia_tests.sh

```diff
@@ -77,6 +77,13 @@ ssh-keygen -y -f $pkey > key.pub
 fuchsia_ctl pave -i $1 --public-key "key.pub"
 echo "$(date) END:PAVING --------------------------------------------"
 
+echo "$(date) START:WAIT_DEVICE_READY -------------------------------"
```

Comment:
Is this the same logic as in the engine?

Reply:
Yes. With the refactor to using a function for fuchsia_ctl, it was easier to diff the two files and see that this was missing.